src/vmspawn/vmspawn.c

```diff
@@ -20,8 +21,10 @@
 #include "gpt.h"
 #include "hexdecoct.h"
 #include "hostname-util.h"
+#include "kernel-image.h"
 #include "log.h"
 #include "machine-credential.h"
+#include "macro-fundamental.h"
```

Comment:
this should not be necessary, is included already in "macro.h", which a good chunk of our headers incldue anyway